core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review.directive.html

```diff
@@ -39,7 +39,10 @@ <h4 class="author-header">Submitted by <[authorName]></h4>
       <textarea class="protractor-test-suggestion-review-message w-100" rows="3" ng-model="reviewMessage"></textarea>
     </div>
   </section>
-  <div ng-show="!reviewable && suggestionIsRejected" class="oppia-suggestion-review-message">
+  <div ng-show="reviewMessage === '' && !reviewable && suggestionIsRejected" class="oppia-suggestion-review-message">
+    <i class="fas fa-spinner fa-spin" aria-hidden="true"></i>
+  </div>
+  <div ng-show="reviewMessage !== '' && !reviewable && suggestionIsRejected" class="oppia-suggestion-review-message">
```

Comment:
Can we use an else block here instead? https://angular.io/api/common/NgIf Ditto below.